SU2_CFD/src/driver_structure.cpp

```diff
@@ -1308,15 +1308,8 @@ void CDriver::Numerics_Preprocessing(CNumerics ****numerics_container,
             default : cout << "Centered scheme not implemented." << endl; exit(EXIT_FAILURE); break;
           }
           
-          if (!config->GetLowFidelitySim()) {
```

Comment:
It is gone... I agree, it is the typical feature that any user would like to try but... if it is not fully operative turns out to be a big source of frustration.